app/src/main/java/de/tum/in/tumcampusapp/activities/generic/ProgressActivity.java

```diff
@@ -51,10 +51,18 @@ public ProgressActivity(int layoutId) {
         super(layoutId);
     }
 
+    public ProgressActivity() { super(); }
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        if (this.mLayoutId != null) {
```

Comment:
This shouldn't be allowed to be null in the first place - thats why we don't have any default constructors.